.gitlab-ci.yml

```diff
@@ -1439,8 +1436,8 @@ deploy_deb_testing-a6:
     - echo "$APT_SIGNING_KEY_ID"
     - printf -- "$APT_SIGNING_PRIVATE_KEY_PART1\n$APT_SIGNING_PRIVATE_KEY_PART2\n" | gpg --import --batch
 
-    - echo "$APT_SIGNING_KEY_PASSPHRASE" | deb-s3 upload -c "pipeline-$DD_PIPELINE_ID" -m 6 -b $DEB_TESTING_S3_BUCKET -a amd64 --sign=$APT_SIGNING_KEY_ID --gpg_options="--passphrase-fd 0 --pinentry-mode loopback --batch --digest-algo SHA512" --preserve_versions --visibility public $OMNIBUS_PACKAGE_DIR/datadog-*_6*amd64.deb
```

Comment:
Is `--pinentry-mode loopback` not supported?

Reply:
Yes, this image has gpg `1.4.16`, `--pinentry-mode` got added in gpg `2.1.12`: https://lists.gnupg.org/pipermail/gnupg-announce/2016q2/000387.html

Reply:
Do you know the implications of changing that? If we don't know, I guess it's okay to try and see: worst case it only affects the staging repo.

Reply:
From the `gpg` manpage:

```
--passphrase-fd n
              Read  the  passphrase  from  file descriptor n. Only the first line
              will be read from file descriptor n.  If  you  use  0  for  n,  the
              passphrase  will  be read from STDIN. This can only be used if only
              one passphrase is supplied.

              Note that since Version 2.0 this passphrase is  only  used  if  the
              option  --batch has also been given. Since Version 2.1 the --pinen-
              try-mode also needs to be set to loopback.
```

Since here we're using gpg `1.4.16`, we don't need `--pinentry-mode` to use  `--passphrase-fd 0` (read from `STDIN`).